flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
Hello.
I have no idea how app will behave on aarch64. It uses git internals, tls, ssh etc.

Reply:
it's clear that we won't know.

Reply:
I will clarify. Remove this.

Reply:
got it, thank you